src/main/scala/is/hail/expr/AnnotationImpex.scala

```diff
@@ -151,7 +151,7 @@ object SparkAnnotationImpex extends AnnotationImpex[DataType, Any] {
             }.toIndexedSeq
         case TGenotype =>
           val g = a.asInstanceOf[Genotype]
-          Row(g.gt.orNull, g.ad.map(_.toSeq).orNull, g.dp.orNull, g.gq.orNull, g.px.map(_.toSeq).orNull, g.fakeRef, g.isDosage)
+          Row(g.gt.orNull, g.ad.map(_.toSeq).orNull, g.dp.orNull, g.gq.orNull, g.px.map(_.toSeq).orNull, g.fakeRef, g.isLinearScale)
```

Comment:
You didn't change the problems here, but the diff did draw my attention to it. This line is _horribly_ inefficient, and people who are using the parquet genotype VDS representation are really going to suffer for it. Can you fix it in a follow-up PR? I'd be curious to see the impact on `write(parquet_genotypes=True)` for profile225 for a fixed version.

Exercise: let's both count the number of allocations in this line.